package.json

```diff
@@ -30,10 +30,12 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "pinia": "^3.0.2",
     "vue": "^3.5.13",
     "vue-router": "^4.5.0"
   },
   "devDependencies": {
+    "@pinia/testing": "^1.0.1",
     "@reduxjs/toolkit": "^2.6.1",
     "@reduxjs/vue-redux": "^1.0.1",
```

Comment:
we could uninstall reduxjs in this branch right?

Reply:
Not yet, because I only converted the config store and left the game store alone